src/AasxServerDB/EntityFW.cs

```diff
@@ -0,0 +1,323 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Extensions;
+using System.ComponentModel.DataAnnotations.Schema;
+
+/*
+ * https://learn.microsoft.com/en-us/ef/core/get-started/overview/first-app?tabs=netcore-cli
+ * 
+ * Initial Migration
+ * Add-Migration InitialCreate -Context SqliteAasContext -OutputDir Migrations\Sqlite
+ * Add-Migration InitialCreate -Context PostgreAasContext -OutputDir Migrations\Postgres
+ * 
+ * Change database
+ * Add-Migration XXX -Context SqliteAasContext
+ * Add-Migration XXX -Context PostgreAasContext
+ * Update-Database -Context SqliteAasContext
+ * Update-Database -Context PostgreAasContext
+ */
+
+namespace AasxServerDB
+{
+    public class AasContext : DbContext
+    {
+        public static IConfiguration _con { get; set; }
+        public static string _dataPath { get; set; }
+        public static bool _isPostgres { get; set; }
+
+        public DbSet<AASXSet> AASXSets { get; set; }
+        public DbSet<AASSet> AASSets { get; set; }
+        public DbSet<SMSet> SMSets { get; set; }

```

Comment:
Don't use abbrevations!
Also **don't** use hungarian notation. And additionaly it can also be misinterpreted as a method that "sets" something.

```suggestion
        public DbSet<SMSet> Submodels { get; set; }

```

Change this for all properties